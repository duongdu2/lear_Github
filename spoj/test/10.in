45
40
6446 6202 665 4695 5586 5994 6013 6631 1101 8575 7890 2155 3764 2685 3880 7432 4305 992 571 8171 6381 4725 3713 8613 9335 387 1037 4866 9484 5856 3835 4486 8211 9478 3998 6897 4816 6049 2600 3185 
10
2197 794 5412 564 5929 1696 9718 4820 9286 2564 
16
2209 5917 6164 9903 4291 3805 483 6292 1032 9630 6067 9691 7094 2108 7713 9224 
49
9584 7128 458 933 9975 1078 4327 4824 2637 7299 5689 6537 9790 1263 5518 1974 1569 2442 8492 1321 3644 3830 4249 6992 9797 6732 8328 2060 240 4365 7501 7798 6787 2757 8104 419 1885 9732 3039 1535 2319 5554 19 4472 609 5022 9222 4668 5067 
4
2067 2267 2612 9037 
18
9484 8517 9900 3249 9457 1772 9010 5837 9066 329 1515 3099 1190 5779 4770 1969 2258 2548 
48
5137 6254 4338 3551 5454 3101 5040 384 8952 7720 3576 1731 7083 6375 3645 2840 3200 1425 9511 8801 4037 187 6771 9158 3510 2403 2984 4935 2024 9659 6626 2635 2673 6008 4744 5408 6928 5217 4428 3018 8575 5963 9740 3987 5155 8771 3580 8603 
27
9290 2510 4319 1939 153 6452 3301 7471 1678 8585 4458 5371 4818 7803 3388 8393 2811 7152 489 6527 1966 9518 7527 3027 8167 6694 33 
9
8580 8158 5140 4861 7096 3653 3292 6533 6310 
5
2885 4005 2224 6588 8541 
26
7470 7150 795 69 7455 6276 3331 5668 5877 300 3037 2923 491 9053 9609 8965 7438 6958 9340 8410 7349 3747 500 9687 9817 9739 
24
8282 3867 8179 6143 8670 5127 1862 1517 2274 3141 9910 8521 6380 5746 77 2998 3866 266 2011 9879 9034 2127 5314 9861 
49
8204 6665 1856 7463 3974 8199 4501 977 8068 3754 117 5481 2109 2487 9792 2005 5522 1924 4963 7839 8212 4814 4183 861 78 2056 6563 6080 8525 1396 8334 6130 37 676 6666 5491 3926 5196 1037 3462 7032 7086 1157 1728 4176 6267 545 4459 9350 
18
360 5434 1008 1960 380 5639 6422 6316 2317 4320 9296 4044 2258 1513 808 8787 9088 9932 
27
48 1165 2349 5979 3015 4995 2328 4459 3640 3952 353 6066 684 9836 5975 117 838 9210 5904 4579 7612 3468 5787 9539 8927 9021 8212 
36
7382 414 331 6513 6089 3742 6713 9466 1754 3223 1206 110 3030 9198 6486 1222 3710 3800 3022 2170 2949 9661 5828 3442 1220 3881 9253 8744 4958 8257 1611 4928 9818 4523 4455 942 
5
8492 4083 2189 9741 3334 
37
3903 5002 5667 3284 7441 3023 4091 9949 8060 5182 4523 3510 6336 6793 7764 4073 9411 4746 1591 8204 8244 7728 5787 4637 2423 2738 3108 5197 7854 1029 3514 8621 2033 7373 3746 735 4894 
47
8192 497 8371 4160 4669 8049 2512 8313 1988 701 7096 9077 4999 9485 1103 3702 5916 7782 7710 7575 4512 9759 7296 9678 137 7092 9584 3007 2706 534 3612 922 9128 2269 8747 5896 2170 6020 2418 4547 8047 6300 4026 6087 6067 8477 7195 
30
1026 8406 2797 8490 2668 7678 3414 9031 3990 9598 2083 6791 2472 2835 5366 9748 912 5443 2741 5271 4225 801 5352 9778 1103 3007 5315 5569 1751 6734 
36
3698 793 7988 2180 8812 4199 7397 6033 9284 7379 6163 6316 2749 4082 300 6251 5214 2594 4450 1036 2366 5642 6943 6111 1467 7193 8231 1707 755 6403 6200 8882 3199 6430 80 7980 
36
6273 9218 1775 6972 3972 4550 2090 2875 4889 4294 439 7015 6646 5415 3790 2698 1445 1125 8792 5728 8894 3635 2659 6886 7589 7154 3716 7896 8233 5018 565 9641 4360 6256 9700 8661 
37
8296 7518 1288 470 9086 265 9913 8823 3081 837 812 3062 7216 7392 8390 8393 4283 8546 6330 8898 3773 8198 9645 5 2541 8513 6229 9750 8538 9974 694 5348 8995 7113 1599 5847 6001 
34
4342 426 2352 9905 4201 3828 6027 2793 4044 8394 9154 2668 3667 4295 9948 4609 8456 5165 6540 3984 1284 1466 5981 6795 511 4143 2800 5920 3194 8611 8313 8382 6403 9358 
14
5705 1924 3539 9272 4015 3917 5858 3407 7967 9258 3754 3069 2070 7307 
38
9718 5408 4256 6723 6381 5824 7364 3768 7174 9352 9803 5631 4190 6348 4651 4206 2629 6575 4199 8121 8223 3952 2708 5344 700 4234 6149 5238 7840 2758 7413 2278 7812 2007 9331 4454 8728 4896 
3
382 6141 9105 
14
5870 35 2279 1103 3638 3055 2993 3276 8003 9711 7794 3378 32 9393 
9
9537 6639 1624 7200 9007 7871 5133 3956 3242 
41
5513 2287 4069 4229 3294 5641 8335 9080 4171 7094 7217 8119 9690 8846 7577 6703 5645 5663 9999 7059 6787 144 9216 3522 749 996 5203 5320 3029 5127 3360 2258 8995 87 1983 3220 8341 9862 5325 428 7290 
28
6076 9551 4662 6256 3775 5551 8392 9668 7552 8291 6364 3611 6803 8610 5733 7648 3999 9546 6867 2924 1358 101 6853 3321 496 3414 6844 9246 
41
161 3067 782 3580 8923 6654 455 6769 2069 3895 6334 1872 1058 5786 2941 4262 1702 9328 3976 6000 8987 693 836 2056 4091 9976 7396 74 633 3034 3201 8627 6430 9599 6253 3298 9058 1191 8677 6029 1346 
30
1319 8745 8351 7678 8622 842 5525 900 582 8647 9672 1239 7981 8824 6820 270 111 4902 6609 736 5611 7251 2391 7422 6713 8608 8760 8352 4481 6378 
5
3386 8750 2691 4931 6389 
6
388 3066 7561 3611 6466 4007 
25
9765 9112 2469 7058 8832 6191 8123 3039 1639 5710 5053 1126 4489 2775 2990 4557 3746 2615 8426 5081 6058 7714 9775 8230 6657 
32
1986 9112 3925 6 2728 68 2382 343 3970 600 8350 9168 569 1026 6856 7689 6088 8694 2349 5938 1524 2812 5353 9055 4727 5575 1640 8874 9662 3777 3189 2923 
27
3882 4513 3908 1856 1352 7899 5858 2794 261 7081 7287 2943 500 9084 585 459 977 7228 2190 7539 4172 9290 447 5174 7051 469 3650 
29
1642 2374 8537 2781 7007 650 8540 8761 9924 5703 2913 680 1522 1229 9791 7698 3093 7269 3731 630 7451 2758 7958 2539 7278 2643 9445 4729 9567 
25
355 2590 3096 3560 8618 2936 9071 9632 1598 3098 7611 6079 7275 6441 8282 8377 6111 9796 9075 9154 5304 8281 482 3883 871 
28
9808 6758 5488 915 8376 7565 445 6222 2898 9980 3466 3289 204 8205 8896 8342 9726 2465 2793 3025 7535 1628 7573 3604 1017 1993 2887 1166 
23
7201 5297 6420 8286 4620 1725 962 9696 7715 1926 3851 3174 6696 8291 9869 1446 1209 7166 5009 4524 4075 7663 8296 
22
3629 9235 2881 6321 3015 9244 2397 1895 7983 6300 4750 4489 5111 9259 4622 1285 2976 7703 535 3422 6799 4406 
4
9117 8010 4688 9163 
8
7684 2919 8142 4486 9609 2873 5117 102 
