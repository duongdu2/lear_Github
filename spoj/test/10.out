28297
24933
29224
29562
13916
28841
28910
27393
23834
19134
29243
29650
27667
28316
28585
28945
22316
27981
29021
29124
26978
28559
29637
29211
26497
28873
15628
27107
26415
29551
28765
28903
27256
20070
18034
28652
27942
26273
29282
28582
29514
27861
27738
26290
25435
